### PR TITLE
Phase2-hgx114 Define a scenario with the new HGCal Geometry

### PR DESCRIPTION
**Pull request by bsunanda**
Put in all the components to make D28 geometry. However the HGCTriggerGeometry needs updating or one should make a scenario without the L1 trigger code

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please check

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/README.md [hunk 1]
```diff
@@ -35,6 +36,7 @@ Several detector combinations have been generated:
 * D25 = T6+C3+M2+I4+O2+F2 
 * D26 = T9+C3+M2+I1+C2+F2
 * D27 = T10+C3+M2+I1+O2+F2
+* D28 = T10+C4+M2+I1+O2+F3
```
Comment (kpedro88) [inline Configuration/Geometry/README.md]: D27 is a test scenario for tracker, so D28 should not be based on that one. I would start from D21 (which is D17 with the T6 tracker update) and just swap C3 with C4. (Also, there is no F3 - a typo?)

File: Configuration/Geometry/python/dict2023Geometry.py [hunk 2]
```diff
@@ -282,11 +282,11 @@
             'Geometry/HcalSimData/data/CaloUtil.xml',
             'Geometry/HGCalSimData/data/hgcsensv8.xml',
             'Geometry/HGCalSimData/data/hgccons.xml',
-            'Geometry/HGCalSimData/data/hgcProdCuts.xml',
         ],
         4 : [
             'Geometry/HcalSimData/data/HcalProdCuts.xml',
             'Geometry/EcalSimData/data/EcalProdCuts.xml',
+            'Geometry/HGCalSimData/data/hgcProdCuts.xml',
```
Comment (kpedro88) [inline Configuration/Geometry/python/dict2023Geometry.py]: if you make this change, you have to rerun the geometry script for all geometries that contain C3.

**Comment by kpedro88** [comment]
@jbsauvan is there a trigger geometry update for the HGCal v9 yet?

**Comment by kpedro88** [comment]
@bsunanda in my opinion, this PR should include new workflows as well as the detector configuration. I can send a PR to your branch creating a new set of workflows.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by jbsauvan** [comment]
Hi @kpedro88 @bsunanda, the trigger geometry has not been adapted to the v9 geometry yet.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+1

**Comment by bsunanda** [comment]
@kpedro88 @ianna @franzoni Can you approve this PR?

**Comment by ianna** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**